java/runtime/src/main/java/io/ray/runtime/AbstractRayRuntime.java

```diff
@@ -51,6 +51,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AbstractRayRuntime.class);
   public static final String PYTHON_INIT_METHOD_NAME = "__init__";
+  public static final String DEFAULT_PLACEMENT_GROUP_NAME = "unnamed_group";
```

Comment:
There's no need to make it public, right?

Reply:
yes,